mockgen/mockgen.go

```diff
@@ -42,6 +42,7 @@ const (
 
 var (
 	source          = flag.String("source", "", "(source mode) Input Go source file; enables source mode.")
+	sourcePackage   = flag.String("source_package", "", "Full package name for interface to be mocked. Required to support references to structs from same package.")
```

Comment:
Can this be figured out automatically from the source rather than needing to be specified on the command line?